build.gradle

```diff
@@ -116,10 +116,18 @@ processResources {
     }
 }
 
+clean {
+    delete minecraft.runDir
+}
+
+runClient.doFirst {
+    copy {
+        from 'src/test/resources/config/templates'
+        into 'run/config/onslaught/templates'
+    }
+}
+
```

Comment:
Hey look at this! the new tasks for gradle to make it less tedious to build and test!